experiments/integrated/fluxnet/fluxnet_simulation.jl

```diff
@@ -6,14 +6,15 @@ N_spinup_days = 30
 N_days = N_spinup_days + 30
 tf = Float64(t0 + 3600 * 24 * N_days)
 t_spinup = Float64(t0 + N_spinup_days * 3600 * 24)
-saveat = Array(t_spinup:(n * dt):tf)
+dt_save = 3600.0
```

Comment:
should we keep `dt_save = n * dt` here?

Reply:
I actually found this more confusing, because every site then has a different n*dt to reach hourly saving.